src/reverb/cron.php

```diff
@@ -8,6 +8,8 @@
  * @package Reverb
  */
 
+header("Access-Control-Allow-Origin: *");
```

Comment:
This is opening you up to XSS attacks. Does this endpoint make sure you are authenticated?

Reply:
Indeed, you are right, I delete this method and I find an other solution with a global variable given by Prestashop in Javascript " base_dir ". It's work better.

thank you